custodial_copy_notifications.tf

```diff
@@ -0,0 +1,54 @@
+locals {
+  custodial_copy_topic_name  = "${local.environment}-dr2-cc-notifications"
+  custodial_copy_topic_arn   = "arn:aws:sns:eu-west-2:${data.aws_caller_identity.current.account_id}:${local.custodial_copy_topic_name}"
+  custodial_copy_queue_name  = "${local.environment}-dr2-cc-notifications"
+  custodial_copy_lambda_name = "${local.environment}-dr2-cc-notifications"
+}
+
+module "dr2_custodial_copy_topic" {
+  source = "git::https://github.com/nationalarchives/da-terraform-modules//sns"
+  sns_policy = templatefile("./templates/sns/custodial_copy_topic_policy.json.tpl", {
+    dr_user_arn = aws_iam_user.disaster_recovery_user.arn
+    sns_topic   = local.custodial_copy_topic_arn
+  })
+  tags       = {}
+  topic_name = local.custodial_copy_topic_name
+}
+
+module "dr2_custodial_copy_queue" {
+  source     = "git::https://github.com/nationalarchives/da-terraform-modules//sqs"
+  queue_name = local.custodial_copy_queue_name
+  sqs_policy = templatefile("./templates/sqs/sns_send_message_policy.json.tpl", {
+    account_id = var.account_number,
+    queue_name = local.custodial_copy_queue_name
+    topic_arn  = local.custodial_copy_topic_arn
+  })
+  encryption_type = "sse"
+}
```

Comment:
The DLQ for this will need adding to the alerting rule. https://github.com/nationalarchives/dr2-terraform-environments/blob/e5d8ecc60592a27374eee8223f871d97f512c76b/common.tf#L341-L363.